File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
import sys
input = sys.stdin.readline


def solution(n:int, m:int) -> list:
    basket = [i for i in range(n+1)]
    for _ in range(m):
        i, j = map(int, input().split())
        basket[i], basket[j] = basket[j], basket[i]  
        
    return basket[1:]


if __name__ == "__main__":
    n,m = map(int, input().split())
    
    
    print(*solution(n,m))

File: node.js/백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const [N, _] = input[0].split(' ').map(Number);
const basket = Array(N+1).fill(0).map((_,i) => i);
input.slice(1).forEach(ball => {
    const [s, e] = ball.split(' ').map(Number);
    if (s !== e) {
        [basket[s], basket[e]] = [basket[e], basket[s]];
    }
});
console.log(basket.slice(1).join(' '));